applications/tuxedo.md

```diff
@@ -0,0 +1,184 @@
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 9 weeks
+- **Full-Time Equivalent (FTE):**  1.5 FTE (Joshy and Andrew will both work roughly three quarters time on this)
+- **Total Costs:** $30,000 (USD)
+
+### Milestone 1 — Tuxedo Core and Cryptocurrency Piece
+
+- **Estimated duration:** 3 weeks
+- **FTE:**  1.5
+- **Costs:** 10,000 USD
+
+Split the existing FRAMEless UTXO project into the generic Tuxedo core, and the first Tuxedo piece which represents a cryptocurrency.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can spin up the example node and transfer tokens |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 1. | Tuxedo Core | We will create the core of the Tuxedo System, analogous to FRAME Executive and FRAME System |
+| 2. | Token Piece | We will create the first Tuxedo piece that serves as a cryptocurrency, analogous to Pallet Balances |
+| 3. | Tuxedo Node Template | We will create a Substrate node with the runtime built with Tuxedo and including the Token piece. Together this will represent a bitcoin-like token (not PoW though, only the token logic is bitcoin-like) |
+
+### Milestone 2 — Wallet and Multisig
+
+- **Estimated Duration:** 3 weeks
+- **FTE:**  1.5
+- **Costs:** 10,000 USD
+
+Create the second Tuxedo piece, and a user-facing wallet
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can spin up the example node and transfer tokens |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 1. | User Wallet | We will create a CLI wallet that users can use to track their tokens in a Tuxedo-based cryptocurrency. This makes the example node actually useable by common users who are curious to explore but not yet ready to dig into the code. |
```

Comment:
Could you add the tech/language you would be using for this?

Reply:
I've added a bit about this. My vision is that the CLI will be written in Rust and communicate directly with the Substrate jsonrpsee endpoint. We won't get much value from projects like [subxt](https://github.com/paritytech/subxt) or [capi](https://github.com/paritytech/capi) because they all assume FRAME and its corresponding metadata.